pkg/blobcache/blobcache.go

```diff
@@ -140,7 +140,7 @@ func (r *blobCacheReference) HasBlob(blobinfo types.BlobInfo) (bool, int64, erro
 			return true, fileInfo.Size(), nil
 		}
 		if !os.IsNotExist(err) {
-			return false, -1, errors.Wrapf(err, "error checking size of %q", filename)
+			return false, -1, err
```

Comment:
Might make sense to leave behind the bit about checking size

Reply:
Fixed